src/graphql.tsx

```diff
@@ -636,6 +636,9 @@ class ObservableQueryRecycler {
     // Stop the query from polling when we recycle. Polling may resume when we
     // reuse it and call `setOptions`.
     observableQuery.stopPolling();
+    observableQuery.setOptions({
+      fetchPolicy: 'cache-only',
```

Comment:
Let’s remove `stopPolling` for `pollInterval: 0` in `setOptions`. It does the same thing: https://github.com/apollographql/apollo-client/blob/1e986c57e12d98f2831616005ef0211d4c9689d1/src/core/ObservableQuery.ts#L307-L311